package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-practical1",
+  "version": "1.0.0",
+  "description": "React app from scratch",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --hot --open --mode development",
+    "build": "webpack --config webpack.config.js --mode production",
+    "format": "prettier --write \"src/**/*.{js,css,html}\""
+  },
+  "keywords": [
+    "React",
+    "Webpack 5",
+    "Starter Template",
+    "Boilerplate"
+  ],
+  "author": "Ritesh Adwani",
+  "license": "ISC",
+  "dependencies": {
+    "@babel/core": "^7.21.4",
```

Comment:
Dependencies that are only useful during development, should be installed as `devDependencies`.